src/main/kotlin/gg/flyte/christmas/listeners/HousekeepingEventListener.kt

```diff
@@ -62,35 +63,36 @@ class HousekeepingEventListener : Listener, PacketListener {
         event<PaperServerListPingEvent> {
             // TODO finish sponsors
             val footer = text("       ")
-                .append("<gradient:#fffdb8:#ffffff>ᴄᴀʀʙᴏɴ.ʜᴏꜱᴛ</gradient>".style())
+                .append("<gradient:${Colors.PURPLE.asHexString()}:${Colors.MAGENTA.asHexString()}>ᴄᴀʀʙᴏɴ.ʜᴏꜱᴛ".style())
                 .append("<grey> • ".style())
                 .append("<gradient:#FF1285:#FA0000>ᴊᴇᴛʙʀ</gradient><gradient:#FA0000:#FF6921>ᴀɪɴs</gradient>".style())
                 .append("<grey> • ".style())
 
             val motd = Component.empty()
                 .append("<b><obf><white>        ||||||  ".style())
-                .append("<gradient:#F396E1:#FFFFFF>ꜰʟʏᴛ</gradient><gradient:#FFFFFF:#FFFFFF>ᴇ</gradient> ".style())
-                .append("<gradient:#51F651:#FAEDCB>ᴄʜʀɪsᴛᴍ</gradient><gradient:#FAEDCB:#D12020>ᴀs ᴇᴠ</gradient><gradient:#D12020:#D12020>ᴇɴᴛ</gradient>".style())
-                .append("<b><obf><white>  ||||||".style())
-                .append("\n".style())
+                .append("<gradient:${Colors.LIGHT_PURPLE.asHexString()}:${Colors.PINK.asHexString()}>ꜰʟʏᴛᴇ ".style())
+                .append("<gradient:${Colors.ORANGE.asHexString()}:${Colors.RED.asHexString()}>ᴄʜʀɪsᴛᴍᴀs ᴇᴠᴇɴᴛ".style())
+                .append("<b><obf><white>  ||||||\n".style())
                 .append(footer)
 
             motd(motd)
             this.maxPlayers = 1
             this.numPlayers = 1
             this.listedPlayers.clear()
-            listOf(
-                "<st><grey>        <reset> ❆ <bold><light_purple>ꜰʟʏᴛᴇ.ɢɢ <red>ᴄʜʀɪsᴛᴍᴀs <green>ᴇᴠᴇɴᴛ <reset><white>❆ <reset><st><grey>         ".style(),
-                "".style(),
-                "            <gold>Join <green><b>now <reset><gold>to play <aqua>x-mas minigames".style(),
-                "              <gold>and support <red>[charity name]".style() // TODO fill charity name and centre
-            ).forEach {
-                this.listedPlayers.add(PaperServerListPingEvent.ListedPlayerInfo(it.toLegacyString().colourise(), UUID.randomUUID()))
+            val serverListPingText = listOf(
+                "<st><grey>        <reset> ❆ <bold><light_purple>ꜰʟʏᴛᴇ.ɢɢ <red>ᴄʜʀɪsᴛᴍᴀs ᴇᴠᴇɴᴛ <reset><white>❆ <reset><st><grey>         ",
+                "  <bold><light_purple>ꜰʟʏᴛᴇ.ɢɢ <grey>• <yellow>ᴄᴀʀʙᴏɴ.ʜᴏꜱᴛ <grey>• <blue>ʙᴜɪʟᴛʙʏʙɪᴛ.ᴄᴏᴍ",
+                "",
+                "            <gold>ᴊᴏɪɴ <green><b>ɴᴏᴡ <reset><gold>ᴛᴏ ᴘʟᴀʏ <aqua>x-ᴍᴀѕ ᴍɪɴɪɢᴀᴍᴇѕ",
+                "              <gold>ᴀɴᴅ ѕᴜᴘᴘᴏʀᴛ <red>[ᴄʜᴀʀɪᴛʏ ɴᴀᴍᴇ]" // TODO fill charity name and centre
+            )
+            serverListPingText.map { MiniMessage.miniMessage().deserialize(it) }.forEach {
```

Comment:
This actually solves this bug, as it now parses it using the default color codes from MiniMessage instead of custom ones.

Reply:
> This actually solves this bug, as it now parses it using the default color codes from MiniMessage instead of custom ones.

What do your custom tag resolvers change? Are they referencing the default Minecraft chat colours, or are they slightly modified based on the palette you sent earlier?

Reply:
This code uses the default MiniMessage instance, so it should only their tag resolvers. They currently do function correctly in-game.